src/screens/DetailGroupImage/index.js

```diff
@@ -7,18 +7,18 @@ import {
   Text,
   View,
   ImageBackground,
-  StatusBar
+  StatusBar,
 } from 'react-native';
 
-import Icon from 'react-native-vector-icons/Octicons';
-import {useNavigation} from '@react-navigation/native';
 import MemoIc_arrow_back_white from '../../assets/arrow/Ic_arrow_back_white';
+import {colors} from '../../utils/colors';
 import {fonts} from '../../utils/fonts';
+import Icon from 'react-native-vector-icons/Octicons';
 import {calculateTime} from '../../utils/time';
 import ModalImageSingleDetail from '../../components/Chat/ModalImageSingleDetail';
-import {COLORS} from '../../utils/theme';
+import {useNavigation} from '@react-navigation/native';
 
-const {width} = Dimensions.get('screen');
+const width = Dimensions.get('screen').width;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The way of getting screen width has been changed. Both ways are correct, but the new one is more explicit which can be beneficial for code readability.
```diff
- const {width} = Dimensions.get('screen');
+ const width = Dimensions.get('screen').width;
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->